app.py

```diff
@@ -1,11 +1,13 @@
 from flask import Flask, request
 import json
+import os
 import werkzeug
 
 from skatt_exceptions import ItemNotFound
 
 #Hent ut skattemeldinger.json
-f = open('skattemeldinger.json',)
+__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+f = open(os.path.join(__location__, 'skattemeldinger.json'))
```

Comment:
for å være sikker på at man åpner fra modulen sitt directory og ikke working dir